documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -65,12 +65,12 @@ spec:
 Cruise Control uses _capacity limits_ to determine if certain resource-based optimization goals are being broken. 
 
 You specify capacity limits for Kafka broker resources in the `brokerCapacity` property in `Kafka.spec.cruiseControl` .
-Capacity limits can be set for the following broker resources in the described units:
+Capacity limits can be set for the following broker resources in the described units (the "i" in the units is optional):
 
-* `disk`            - Disk storage in bytes
-* `cpuUtilization`  - CPU utilization as a percent (0-100)
-* `inboundNetwork`  - Inbound network throughput in bytes per second
-* `outboundNetwork` - Outbound network throughput in bytes per second
+* `disk`            - Disk storage per broker in megabytes (Mi), gigabytes (Gi) or terabytes (Ti) (Default: 100000Mi)
```

Comment:
The example bellow shows `100G`. Should we change it to `100Gi`? Or add `G` here as unit as well?

Reply:
I left the "i" out of the examples on purpose to highlight the "i" is optional. But I admit that this might be confusing?